core/src/main/java/org/apache/gravitino/metrics/source/MetricsSource.java

```diff
@@ -41,6 +41,7 @@ public abstract class MetricsSource {
   public static final String ICEBERG_REST_SERVER_METRIC_NAME = "iceberg-rest-server";
   public static final String GRAVITINO_SERVER_METRIC_NAME = "gravitino-server";
   public static final String JVM_METRIC_NAME = "jvm";
+  public static final String RELATION_DATASOURCE_METRIC_NAME = "relation-datasource";
```

Comment:
`relation-datasource` seems confusing, I may wonder what's it in Gravitino, is it the relation entity store in Gravitino? I prefer to using `gravitino-entityStore` as the metrics name since it's this is the metrics for entity store and relation datasource is one of the implementation, we you place relation information in the metrics name like `relationDatasource. activeConnectionCount`,   WDYT? cc @yuqi1129

Reply:
> I prefer to using entity-store.relation-datasource. active-connection-count as the metrics name, WDYT? 

I totally agree, the name `entity-store.relation-datasource. active-connection-count` or similar is more clear as some catalogs like `MySQL` or `Doris` catalogs also use JDBC datasouce pool, it's meaningful to distinguish them.

Reply:
syntax wise, I'd suggest using 'active-connections' rather than 'active-connection-count'.